scripts/cbc_syslog_forwarder

```diff
@@ -19,7 +19,7 @@ import logging.handlers
 import psutil
 import sys
 
-from cbc_syslog import poll, check, history
+from cbc_syslog import poll, check, history, wizard
```

Comment:
Feedback on the history function - apologies that it's late.
* It's not clear what is meant to be provided for "source".  In the wizard it prompts for org name to use in the [header] with url, api id etc.
* Would be nice if the help gave an example timestamp in the format.

Reply:
I am going to correct to source name in the wizard as I don't really want it to be mentally associated with the CBC org name.

Reply:
Changed all org name references to source name and provided a sample timestamp